src/GitLink/Extensions/PdbExtensions.cs

```diff
@@ -68,8 +68,15 @@ public static IEnumerable<string> FindMissingOrChangedSourceFiles(this PdbFile p
 
         private static bool AreEqualBuffers(byte[] first, byte[] second)
         {
-            Argument.IsNotNull(nameof(first), first);
-            Argument.IsNotNull(nameof(second), second);
+            if (first == null && second == null)
```

Comment:
Just as a discussion, want to know your thoughts, I am always struggling with this. Why not use ReferenceEquals(null, first) here? No need to get into a (possibly expensive, we don't know the implementation here) == operation just to check for null?